### PR TITLE
oAuth token are setup in the -s flag and saved in config.yml

### PR DESCRIPTION
**Pull request by xnormand**
Can post to group only with -g flag

### CODE REVIEW

**Comment by joejwright** [comment]
Do you think it is necessary to have users register an app with yammer to use the gem? I think most people won't want to do that. Maybe there should be a different cli option to set up those parameters instead of including them in the setup option. What do you think?

**Comment by xnormand** [comment]
Correct me if i'm wrong, to be able to post a status update on a yammer account, the user must authorize the app. Maybe a better way to save those information would be in a hidden file in the home folder of the user. What do you think?

**Comment by joejwright** [comment]
The way oAuth works is that you have to register the client application (yammer-cli) with the web application (yammer). When you register the app you get the consumer token and secret. This doesn't change per user.

When you run the yammer-cli with the -s switch it uses those parameters to bring up the yammer authorization web page. Once you authorize yammer-cli, yammer gives us the oauth token and oauth secret which yammer-cli then uses to communicate with yammer.

So basically the consumer token and secret should never change. The token and secret are the only settings that are user specific.

I probably didn't explain this too clearly but hopefully you get the idea.

This is where the oAuth token and secret come from: https://www.yammer.com/client_applications

**Comment by xnormand** [comment]
When i try to authorize your app using consumer token and secret in the gem, yammer reply that i'm not authorized to user that app and i can't find it anywere in the app directory. Is your app public or private?

I'm so sorry about all that this is why i didn't understand whole thing correctly.

so i will rewrite my code to only save the oAuth token and secret in the user home directory and leave to app token in the gem code.

What do you think?

**Comment by joejwright** [comment]
I was just about to test that same thing this morning. I don't think the app is public yet. I think making it public would make it a lot easier for everyone to use since casual users probably won't want to add the app to your account.

I'll let you know how much progress I make on that.
